library/change_network_type.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/python
+
+from ansible.module_utils.basic import AnsibleModule
+import time
+import subprocess
+
+
+def run_command(command):
+    """Run a shell command and return its output or raise an error."""
+    try:
+        result = subprocess.run(
+            command, shell=True, text=True, capture_output=True, check=True
+        )
+        return result.stdout.strip(), None
+    except subprocess.CalledProcessError as err:
+        return None, Exception(f"Command '{command}' failed: {err.stderr.strip()}")
+
+
+def main():
+    module_args = dict(
+        network_type=dict(type="str", required=True),  # Target network type
```

Comment:
I think ansible supports `choice` type parameters.

```python
    network_type=dict(
        type="str",
        choices=["OVNKubernetes", "OpenShiftSDN"], 
        required=True
    ),
```

maybe

Reply:
yes I have already used it in trigger_network_type. Adding these validations is one of the story.

Reply:
done.